src/secureCreatives.js

```diff
@@ -53,9 +53,12 @@ function receiveMessage(ev) {
 }
 
 function sendAdToCreative(adObject, remoteDomain, source) {
-  const { adId, ad, adUrl, width, height } = adObject;
-
-  if (adId) {
+  const { adId, ad, adUrl, width, height, renderer } = adObject;
+  // rendering for outstream safeframe
+  if (renderer && renderer.url) {
+    const bid = auctionManager.findBidByAdId(adId);
```

Comment:
This `bid` value is the exact same as the `adObject` that gets passed into the function; so this may not be needed.